labgrid/remote/client.py

```diff
@@ -740,6 +740,12 @@ def fastboot(self):
             drv = AndroidFastbootDriver(target, name=None)
         drv.fastboot.timeout = self.args.wait
         target.activate(drv)
+        if args[0] == 'flash':
+            drv.flash(args[1], args[2])
+            return
+        if args[0] == 'boot':
+            drv.boot(args[1])
+            return
```

Comment:
Is there any particular reason for comparing args[0] with 'boot' and 'flash' here again? Can we move getting the driver to start of the method and use the existing ifs instead of comparing again?

Reply:
I'd prefer to check the arguments before doing things that might fail for other reasons.